auslib/db.py

```diff
@@ -946,7 +946,22 @@ def getReleaseInfo(self, product=None, limit=None,
             column = [self.name]
         else:
             column = [self.name, self.product, self.data_version, self.read_only]
+
         rows = self.select(where=where, columns=column, limit=limit, transaction=transaction)
+        j = join(dbo.releases.t, dbo.rules.t, dbo.releases.name == dbo.rules.mapping)
+        ref_list = select([dbo.releases.name, dbo.rules.rule_id]).select_from(j).execute().fetchall()
+        ref_dict = {}
+
+        for ref in ref_list:
+          rel_name, rule_id = ref
+          try:
+            ref_dict[rel_name].append(rule_id)
+          except KeyError:
+            ref_dict[rel_name] = [rule_id]
+
+        for row in rows:
+          row['rule_ids'] = ref_dict[row['name']]
+
```

Comment:
Looks like there's at least one issue with this code that the test is catching. Have a look at the log on https://tools.taskcluster.net/task-graph-inspector/#CuxnWYRESRm-t8rmHEPpuw/YJHyOs7lQGGmoQ0v8F_u_g/0 for more details.

You'll definitely need to update the tests because you're changing the format of the return value here. Please also add tests to make sure that the new functionality works correctly.